ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -50,23 +50,28 @@ tests:
       ODF_CONSOLE_FULL_IMG_NAME: odf-console
     env:
       COMPUTE_NODE_TYPE: m4.2xlarge
-      ZONES_COUNT: "1"
+      ZONES_COUNT: "3"
     test:
     - as: tests
       cli: latest
-      commands: ./test-prow-e2e.sh ${ODF_CONSOLE_FULL_IMG_NAME};
+      commands: |
+        export PULL_SECRET="$(cat /etc/odf-console-e2e/odf-e2e-credentials/config-pull-secret)"
+        ./test-prow-e2e.sh ${ODF_CONSOLE_FULL_IMG_NAME}
       credentials:
       - mount_path: /var/run/operator-secret
         name: operator-secret
         namespace: test-credentials
+      - mount_path: /etc/odf-console-e2e/odf-e2e-credentials
+        name: odf-e2e-credentials
+        namespace: test-credentials
       dependencies:
       - env: ODF_CONSOLE_FULL_IMG_NAME
         name: odf-console-img
       from: src
       resources:
         requests:
-          cpu: 100m
-          memory: 200Mi
+          cpu: 1000m
+          memory: 1Gi
```

Comment:
This is only increasing the memory and CPU of the pod used to run cypress on. This is not required.

Reply:
ACK.

Reply:
yeah, I was trying out few things earlier, missed removing it later...